spec/cypress/integration/admin/DeliveryPartnerManagement.feature

```diff
@@ -0,0 +1,112 @@
+Feature: Admin user modifying delivery partners
+  Admin users should be able to create and delete delivery partners
+
+  Background:
+    Given I am logged in as an "admin"
+    And scenario "admin/suppliers" has been run
+    And I am on "delivery partner index" page
+
+  Scenario: Creating a new delivery partner
+    When I click on "create delivery partner button"
+    Then I should be on "choose new delivery partner name" page
+    And the page should be accessible
+
+    When I type "New delivery partner" into "delivery partner name" field
+    And I click the submit button
+    Then I should be on "choose new delivery partner lead providers" page
+    And the page should be accessible
+
+    When I click on "Lead Provider 1" label
+    And I click the submit button
+    Then I should be on "choose delivery partner cohorts" page
+    And the page should be accessible
+
+    When I click on "2021" label
+    And I click the submit button
+    Then I should be on "new delivery partner review" page
+    And "main" should contain "New delivery partner"
+    And "main" should contain "Lead Provider 1"
+    And "main" should contain "2021"
+    And the page should be accessible
+
+    When I click the submit button
+    Then I should be on "delivery partner index" page
+    And "main" should contain "New delivery partner"
+    And "notification banner" should contain "Delivery partner created"
+    And the page should be accessible
+
+  Scenario: It should remember details when navigating backwards in creation process
+    When I click on "create delivery partner button"
+    And I type "New delivery partner" into "delivery partner name" field
+    And I click the submit button
+    And I click the back link
+    Then "delivery partner name" should have value "New delivery partner"
+
+    When I click the submit button
+    And I click on "Lead Provider 1" label
+    And I click the submit button
+    And I click on "2021" label
+    And I click the submit button
+    And I click the back link
+    Then "Lead Provider 1" label should be checked
+
+    When I click the submit button
+    Then "2021" label should be checked
+
+  Scenario: It should allow changing name choice during creation
+    When I click on "create delivery partner button"
+    And I type "wrong name" into "delivery partner name" field
+    And I click the submit button
+    And I click on "Lead Provider 1" label
+    And I click the submit button
+    And I click on "2021" label
+    And I click the submit button
+    And I click on "change name link"
+    And I type "{selectall}New delivery partner" into "delivery partner name" field
```

Comment:
> I type "{selectall}New delivery partner"

I don't expect to be typing `{selectall}` into any field. What does it do? And why?

Reply:
It's a bit of a weird syntax, but I do this all the time when using websites! It's the equivalent of pressing cmd+a to replace the contents of an input

Reply:
Soo, it's like deleting all content and entering what we want?

It there some way of writing it in more straightforward english? Like "I delete contents of ... field And I type ... into ... field"?

Reply:
We could add "And I clear the input" as a separate step but it seems a bit strange to me - personally I don't think of clearing and then typing in an input as two separate steps, I think of replacing the contents of an input as a single step

Also, there's other stuff like `{enter}` which is useful as users do that too - but definitely doesn't make sense as a separate step to me

Reply:
I think by using braces and syntax that doesn't sound like spoken english we lose some of the benefits we think cucumber will provide - namely the documentation that's understandable to a non-technical person (be it product managers, content designers, testers, anyone else who wants to know). 

Could we make `And I type` clear input before typing by default?

And another `And I type {string} into {string} field without clearing`?

Reply:
That feels a bit magic to me and not the behaviour I would expect

Reply:
Does `I type {string} into {string} field after clearing it` sound less magic?

Reply:
Sure, but it violates cucumber best practice which says anything like `after` and `and` in a step should be two separate steps instead 😅 

I feel like the easiest way to resolve this would be to just make it two separate steps

Reply:
I am more happy with two steps than with `{selectall}`.

I guess `I replace content of {string} with {string}` might be a more natural way of describing it, but I'm not sure that fits the style of our descriptions. 

Arghhh, why is english so hard.